pkg/attestation/crafter/runners/githubaction.go

```diff
@@ -68,3 +100,28 @@ func (r *GitHubAction) RunURI() (url string) {
 func (r *GitHubAction) ResolveEnvVars() (map[string]string, []*error) {
 	return resolveEnvVars(r.ListEnvVars())
 }
+
+func (r *GitHubAction) Environment() RunnerEnvironment {
+	if r.githubToken != nil {
+		switch r.githubToken.RunnerEnvironment {
+		case "github-hosted":
+			return Managed
+		case "self-hosted":
+			return SelfHosted
+		default:
+			return Unknown
+		}
+	}
+	return Unknown
+}
+
+func (r *GitHubAction) WorkflowFilePath() string {
+	if r.githubToken != nil {
+		return r.githubToken.JobWorkflowRef
+	}
+	return ""
```

Comment:
check https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables for the default values for GITHUB_WORKFLOW_REF and RUNNER_ENVIRONMENT

Reply:
as mentioned, it seem those env are not available for each run, at least I don't see them present